package.json

```diff
@@ -7,8 +7,15 @@
     "Editor",
     "WYSIWYG"
   ],
-  "dependencies": {},
+  "dependencies": {
+    "ckeditor5-core": "ckeditor/ckeditor5-core"
```

Comment:
Pick the one you prefer:

A1: [RTFM](https://docs.npmjs.com/files/package.json#github-urls)! (just kidding... :D)

A2: That's the way to point npm to GitHub repos: `organization-name/repo-name` ;)